extensions/adobe/experience/offer-management/fallback-content-offer.schema.json

```diff
@@ -29,6 +26,9 @@
     {
       "$ref": "https://ns.adobe.com/xdm/data/record"
     },
+    {
+      "$ref": "https://ns.adobe.com/experience/decisioning/option"
+    },
```

Comment:
This mixin is in the meta:extends list, so it should be in the allOf list as well